ModularTegustation/tegu_traitor_code/velvet_fu.dm

```diff
@@ -0,0 +1,262 @@
+	A.adjustStaminaLoss(40)
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, TRUE, -1)
+	return TRUE
+
+//Full Thrust - Deals Brute and has a chance to knock the opponent down. Costs 50 Stamina.
+/datum/martial_art/velvetfu/proc/fullThrust(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	log_combat(A, D, "full thrust (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] thrusted into [D]!</span>", \
+					"<span class='userdanger'>You're thrusted by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You quickly and fashionably thrust into [D]!</span>")
+	A.adjustStaminaLoss(40)
+	D.apply_damage(20, BRUTE)
+	if(prob(50))
+		D.Knockdown(20)
+	playsound(get_turf(A), 'sound/weapons/cqchit1.ogg', 50, TRUE, -1)
+	return TRUE
+
+//Minor Iris - Deals a ton of armor penetrating slash brute. Costs 80 Stamina.
+/datum/martial_art/velvetfu/proc/minorIris(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	log_combat(A, D, "minor iris (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] slashes [D] rapidly and repeatedly!</span>", \
+					"<span class='userdanger'>You're slashed several times by [A]!</span>", "<span class='hear'>You hear several sickening sounds of flesh slashing flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You swiftly and repteadly slash at [D], truly a master attack!</span>")
+	A.adjustStaminaLoss(80)
+	D.apply_damage(50, BRUTE)
+	var/obj/item/bodypart/bodypart = pick(D.bodyparts)
+	var/datum/wound/slash/severe/crit_wound = new
+	crit_wound.apply_wound(bodypart)
+	playsound(get_turf(A), 'sound/weapons/bladeslice.ogg', 50, TRUE, -1)
+	return TRUE
+
+/datum/martial_art/velvetfu/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	add_to_streak("H",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "harmed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = pick("silken wrist")
+	var/bonus_damage = 10
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 5
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, BRUTE)
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/datum/martial_art/velvetfu/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(!can_use(A))
+		return FALSE
+	add_to_streak("G",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "grabbed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = pick("ascending claw")
```

Comment:
If you aren't planning on adding more text here, there is no need for pick() at all.